databook/physiology/stimuli/passive-visual-stimuli/visual-stimuli-list.md

```diff
@@ -0,0 +1,48 @@
+# Visual Stimuli
+
+# Probably want figures of the stimuli
+
```

Comment:
Hold off on this - if the idea is not to use the vc2p stimulus page this needs to be much more generic. It's fine to leave this - we can edit it later.